src/renderer/common/js/utils.js

```diff
@@ -42,6 +44,24 @@
         shell.openExternal(url);
     };
 
+
+    /**
+     * Executes the file from the shareXPath with the combination of the command
+     * @param command {String} - shareX command, example: "Morphic: Capture entire screen to desktop"
+     * @param shareXPath {String} - the path and executable name, example: "C:\\sharex-portable\\sharex.exe"
+     */
+    gpii.psp.execShareXCommand = function (command, shareXPath) {
+        // creates the command line, it should looks something like:
+        // "C:\\sharex-portable\\sharex.exe" -workflow "Morphic: Capture entire screen to desktop"
+        let commandToExecute = "\""+shareXPath+"\" -workflow \""+command+"\"";
+
+        try {
+            child_process.exec(commandToExecute);
```

Comment:
Should return something - e.g. a promise for the completion of the command

Reply:
Done